warden/x/async/keeper/genesis.go

```diff
@@ -29,11 +29,16 @@ func (k *Keeper) ExportState(ctx sdk.Context, genState *types.GenesisState) erro
 }
 
 func (k *Keeper) AddProphetPlugins(ctx sdk.Context) error {
+	commonPluginFees := types.PluginFees{
```

Comment:
i think it's clearer to call it "zero fees"

```suggestion
	zeroFees := types.PluginFees{
```